keyboards/fleuron/fleuron_fullgrid_numpad_right.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "Fleuron v1.0", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "bootloader": "", 
+    "width": 16, 
+    "height": 6, 
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This MUST match the name from the `fleuron.h` file  (eg, it needs to be `KEYMAP` or `LAYOUT_ortho_6x16`)

Reply:
Should this filename be something else, like info.json, or fleuron.json?

Reply:
Done :)